cmd/desync/cat.go

```diff
@@ -32,6 +32,8 @@ func cat(ctx context.Context, args []string) error {
 		offset         int
 		length         int
 		readIndex      bool
+		clientCert string
```

Comment:
Minor stylistic quibble, but it'd be good for the type name (`string`) to line up with the `bool` from the lines above.